### DISCUSSION TITLE
AWS - AMI Marketplace Install

### DISCUSSION THREAD

**Discussion by malibali** [Security-Onion-Solutions/securityonion#14042] [Unsupported Versions] [answered]
Hello,

I'm using the following AMI to install SO2: https://aws.amazon.com/marketplace/pp/prodview-4gpqv3qlxq4ww?ref=_ptnr_soc_docs_230525 

Everything is going great until the script tries to connect to sigs.securityonion.net and then it just hangs and fails to connect.

Thoughts?

Austin

**Comment by jertel** [comment, marked as answer]
Hello! Are you able to access (via curl) other Internet web hostnames from within the VM? If not, check the VM's outbound Internet connectivity / security group rules.

**Comment by malibali** [comment]
Thanks for the follow up!  So it turned out that by default, only SSH (port 22) was allowed with the AMI.  Once I allowed 443/80 things progressed.  Having a Windows background, it took me awhile to figure this out after you suggestion.
